fetch/api/request/request-init-stream.any.js

```diff
@@ -2,56 +2,146 @@
 
 "use strict";
 
+const duplex = "half";
+const method = "POST";
+
 test(() => {
-  const stream = new ReadableStream();
-  const request = new Request("...", { method:"POST", body: stream });
-  assert_equals(request.body, stream);
+  const body = new ReadableStream();
+  const request = new Request("...", { method, body, duplex });
+  assert_equals(request.body, body);
 }, "Constructing a Request with a stream holds the original object.");
 
-async function assert_request(test, input, init) {
-  assert_throws_js(TypeError, () => new Request(input, init), "new Request()");
-  await promise_rejects_js(test, TypeError, fetch(input, init), "fetch()");
-}
```

Comment:
Why remove this?

(If there's a good reason to remove this, presumably we don't need to use `promise_test` below?)